lib/drivers/tandemDriver.js

```diff
@@ -1482,7 +1482,7 @@ module.exports = function (config) {
 
       // bolus records
       if (bolus.bolex_size !== undefined || bolus.bolus_option === 'extended') {
-        if (bolus.bolus_size !== undefined || bolus.bolus_insulin_requested !== undefined) {
+        if (bolus.bolus_size !== undefined || bolus.insulin_requested !== undefined) {
```

Comment:
Yes, `bolus_size` only refers to normal portion, and that's how I made sense of it as well.